tests/kuttl/tests/change_keystone_config/02-assert.yaml

```diff
@@ -0,0 +1,6 @@
+# check that by default, debug is not set in keystone config
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+namespaced: true
+commands:
+  - script: $KEYSTONE_KUTTL_DIR/../common/scripts/check_debug_in_keystone_pod_logs.sh --reverse
```

Comment:
Don't we deploy the `KeystoneAPI` sample with `debug = true` to start [1]?  Or are we doing some other manipulation to remove debug initially?

https://github.com/openstack-k8s-operators/keystone-operator/blob/d6671c8d0e22e4b6d4b4350223d54554a2980775/config/samples/keystone_v1beta1_keystoneapi.yaml#L10-L12

Reply:
Yes, and that is written to `/etc/keystone/keystone.conf.d/custom.conf`, but it seems that is not applied by default, since the logs do not show debug messages.

Reply:
I feel like you found some bugs :)

Reply:
I think we're keeping everything in Jira, so let's add a bug there